instrumentation/jmx-metrics/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/jmx/JmxMetricInsightInstaller.java

```diff
@@ -81,10 +80,15 @@ private static void buildFromDefaultRules(
   private static void buildFromUserRules(
       MetricConfiguration conf, ConfigProperties configProperties) {
     List<String> configFiles = configProperties.getList("otel.jmx.config");
+    String configFileUrlString;
     for (String configFile : configFiles) {
-      JmxMetricInsight.getLogger().log(FINE, "JMX config file name: {0}", configFile);
+      configFileUrlString = configFile;
+      if (configFile != null && !configFile.contains("://")) {
+        configFileUrlString = "file://" + configFile;
```

Comment:
So the assumption here is that user provides a properly encoded url, but the user might also provide a path to a file  which isn't really a properly encoded url (e.g the space in `/tmp/foo bar` should be encoded as `%20` in url). Perhaps there should be a comment that explains why the file path doesn't need to be encoded.

Reply:
Also, there's an issue if the user specified filepath is relative.

Reply:
@PeterF778 fixed